### PR TITLE
Shooter update for Final Bot

### PR DESCRIPTION
**Pull request by ChaoticChaosTheGreat**
added the other two motors including a 
- follower motor
- feeder motor

### CODE REVIEW

**Comment by ChaoticChaosTheGreat** [comment]
Cannot show it working because swerve sim not working

File: src/main/java/frc/robot/subsystems/swerve/commands/ZeroGyro.java
Comment (realjoshuau) [inline src/main/java/frc/robot/subsystems/swerve/commands/ZeroGyro.java:1]: What

File: src/main/java/frc/robot/subsystems/shooter/commands/ShootSpeaker.java [hunk 1]
```diff
@@ -35,6 +36,6 @@ public void end(boolean interrupted) {
 
   @Override
   public boolean isFinished() {
-    return Math.abs(shooterSubsystem.getRps() - rps) < 0.1;
+    return false;
```
Comment (realjoshuau) [inline src/main/java/frc/robot/subsystems/shooter/commands/ShootSpeaker.java:39]: huh

**Review by realjoshuau — APPROVED** [review]
lgtm

**Pull request merged**